### PR TITLE
[13.0][MIG] lunch

### PR DESCRIPTION
**Pull request by Yasaie**
Migration of `lunch`
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/lunch/migrations/13.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ('lunch.alert ', 'lunch_alert ', 'specific_day', 'until'),
+]
+
+_field_adds = [
+    ("category_id", "lunch.order", "lunch_order", "many2one", False, "lunch"),
+    ("product_id", "lunch.order", "lunch_order", "many2one", False, "lunch"),
+    ("supplier_id", "lunch.order", "lunch_order", "many2one", False, "lunch"),
+    ("price", "lunch.order", "lunch_order", "float", False, "lunch"),
+    ("note", "lunch.order", "lunch_order", "text", False, "lunch"),
+    ("state", "lunch.order", "lunch_order", "selection", False, "lunch"),
```
Comment (MiquelRForgeFlow) [inline addons/lunch/migrations/13.0.1.0/pre-migration.py]: ERROR: column "state" of relation "lunch_order" already exists

File: addons/lunch/migrations/13.0.1.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+]
+
+
+def move_lunch_order_line_to_lunch_order(env):
+    openupgrade.add_fields(env, _field_adds)
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE lunch_order lo
+        SET 
+            category_id = lol.category_id,
```
Comment (MiquelRForgeFlow) [inline addons/lunch/migrations/13.0.1.0/pre-migration.py]:
```suggestion
        SET
            category_id = lol.category_id,
```
flake8

File: addons/lunch/migrations/13.0.1.0/pre-migration.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+def move_lunch_order_line_to_lunch_order(env):
+    openupgrade.add_fields(env, _field_adds)
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE lunch_order lo
+        SET 
+            category_id = lol.category_id, 
+            product_id = lol.product_id,
+            supplier_id = lol.supplier_id,
```
Comment (MiquelRForgeFlow) [inline addons/lunch/migrations/13.0.1.0/pre-migration.py]:
```suggestion
            supplier_id = lol.supplier,
```

File: addons/lunch/migrations/13.0.1.0/post-migration.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_company_id(cr):
+    table_list = [
+        'lunch.order',
+        'lunch.product.category',
```
Comment (MiquelRForgeFlow) [inline addons/lunch/migrations/13.0.1.0/post-migration.py]:
```suggestion
        'lunch_order',
        'lunch_product_category',
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by Yasaie** [comment]
@pedrobaeza Re-based to the latest

**Comment by pedrobaeza** [comment]
OK, merging after the approval of Miquel. Do you really use this module?

**Pull request merged**